File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/OuterStructWithFieldConstructor.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class OuterStructWithFieldConstructor {
    @NonNull
    public OuterStructWithFieldConstructor.InnerStructWithDefaults outerStructField;
    public static final class InnerStructWithDefaults {
        public double innerStructField;
        public InnerStructWithDefaults() {
            this.innerStructField = 1.0;
        }
    }
    public OuterStructWithFieldConstructor(@NonNull final OuterStructWithFieldConstructor.InnerStructWithDefaults outerStructField) {
        this.outerStructField = outerStructField;
    }
}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/OuterStructWithFieldConstructor.kt
/*

 *
 */

@file:JvmName("OuterStructWithFieldConstructor")

package com.example.smoke


class OuterStructWithFieldConstructor {
    @JvmField var outerStructField: OuterStructWithFieldConstructor.InnerStructWithDefaults

    class InnerStructWithDefaults {
        @JvmField var innerStructField: Double



        constructor() {
            this.innerStructField = 1.0
        }





    }




    constructor(outerStructField: OuterStructWithFieldConstructor.InnerStructWithDefaults) {
        this.outerStructField = outerStructField
    }





}

